src/ports/postgres/modules/deep_learning/madlib_keras.sql_in

```diff
@@ -158,12 +161,13 @@ CREATE AGGREGATE MADLIB_SCHEMA.fit_step(
     /* architecture */           TEXT,
     /* compile_params */         TEXT,
     /* fit_params */             TEXT,
-    /* use_gpu */                BOOLEAN,
+    /* gpus_per_host  */         INTEGER,
+    /* segments_per_host  */     INTEGER,
     /* previous_state */         BYTEA
 )(
     STYPE=BYTEA,
     SFUNC=MADLIB_SCHEMA.fit_transition,
-    m4_ifdef(`__POSTGRESQL__', `', `prefunc=MADLIB_SCHEMA.fit_merge,')
```

Comment:
I'm not sure if this was here before and I missed it, or if it's new.  Looks like this would break postgres support, but I haven't tested it--was this change intentional?

Reply:
Turns out the postgres throws a warning saying `aggregate attribute "prefunc" not recognized` but does not fail. I think it's safer to have the if check so i will revert this to the original m4 ifdef.